xcm/xcm-executor/src/traits/conversion.rs

```diff
@@ -24,6 +24,9 @@ use xcm::v0::{MultiLocation, OriginKind};
 /// One of `convert`/`convert_ref` and `reverse`/`reverse_ref` MUST be implemented. If possible, implement
 /// `convert_ref`, since this will never result in a clone. Use `convert` when you definitely need to consume
 /// the source value.
+///
+/// Can be amalgamated into tuples. If any of the tuple elements converts into `Ok(_)` short circuits, otherwise returns
+/// the `Err(_)` of the last failing conversion (or `Err(())` for ref conversions).
```

Comment:
```suggestion
/// Can be amalgamated into tuples. If any of the tuple elements converts into `Ok(_)` it short circuits. Otherwise returns
/// the `Err(_)` of the last failing conversion (or `Err(())` for ref conversions).
```